ui/app/models/pki/pki-issuer-engine.js

```diff
@@ -13,7 +13,7 @@ const validations = {
 };
 
 @withModelValidations(validations)
-export default class PkiIssuersEngineModel extends Model {
+export default class PkiIssuerEngineModel extends Model {
```

Comment:
trying to make things consistent with the model class names. There is some inconsistency with PKI and in the API with plural vs. non plural. Doing my best to be consistent.